Drivers/seds/UartHwasExample/linuxPartition/main.cpp

```diff
@@ -0,0 +1,61 @@
+/**@file
```

Comment:
I remember looking at this file.... Are you sure that everything is properly updated and targetted?

Reply:
I will check that. 
Here it should be indicated that this file was renamed, not created.

Reply:
done